From 5d48a53e45aa5ddc46e88a0be50b5cc6152d6d6c Mon Sep 17 00:00:00 2001
From: Geert Uytterhoeven <geert+renesas@glider.be>
Date: Wed, 14 May 2014 16:41:12 +0200
Subject: ARM: shmobile: r8a7740 dtsi: Remove duplicate interrupt-parent
 property

Caused by interaction between commit
08ec67b50db7ca8c9077e67ca23850cdc5bfc716 ("ARM: shmobile: r8a7740 dtsi:
Add Ethernet support") and commit 9ff254adc1e32db46000a33b8ecbc4d7047672be
("ARM: shmobile: dts: Move interrupt-parent property to root node").

Signed-off-by: Geert Uytterhoeven <geert+renesas@glider.be>
Acked-by: Laurent Pinchart <laurent.pinchart@ideasonboard.com>
Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
(cherry picked from commit 3fa3985e624aea24334abc9a33b484c3d316e64e)
Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
---
 arch/arm/boot/dts/r8a7740.dtsi | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/boot/dts/r8a7740.dtsi b/arch/arm/boot/dts/r8a7740.dtsi
index ce7a0b29ae7c..55d29f4d2ed6 100644
--- a/arch/arm/boot/dts/r8a7740.dtsi
+++ b/arch/arm/boot/dts/r8a7740.dtsi
@@ -124,7 +124,6 @@
 		compatible = "renesas,gether-r8a7740";
 		reg = <0xe9a00000 0x800>,
 		      <0xe9a01800 0x800>;
-		interrupt-parent = <&gic>;
 		interrupts = <0 110 IRQ_TYPE_LEVEL_HIGH>;
 		/* clocks = <&mstp3_clks R8A7740_CLK_GETHER>; */
 		phy-mode = "mii";
-- 
2.1.2

